apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.sql

```diff
@@ -1,16 +1,16 @@
 -- BLOCK select_generation_sequence_by_course
 SELECT
-  to_jsonb(j.*) AS job,
+  to_jsonb(js.*) AS job_sequence,
   to_jsonb(u.*) AS user
 FROM
-  jobs AS j
-  LEFT JOIN users AS u ON (j.authn_user_id = u.user_id)
+  job_sequences AS js
+  LEFT JOIN users AS u ON (js.authn_user_id = u.user_id)
 WHERE
-  j.course_id = $course_id
+  js.course_id = $course_id
   AND (
-    j.type = 'ai_question_generate'
-    OR j.type = 'ai_question_regenerate'
+    js.type = 'ai_question_generate'
+    OR js.type = 'ai_question_regenerate'
   )
```

Comment:
Nit:
```suggestion
  AND js.type IN ('ai_question_generate', 'ai_question_regenerate')
```

There is currently no index for `type`, but if this index is eventually added this construct would allow the index to be used. Besides, this is cleaner.

Reply:
thanks for the suggestion- done in 2482afa5493c3b590f2f7aa967093eced91df73d